rrule-wasm/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+use rrule::RRuleSet;
+use wasm_bindgen::prelude::*;
+
+/**
+  Get all recurrences of the rrule!
+     rule_set_string: List of rrules
+     limit: Limit must be set in order to prevent infinite loops
+*/
+#[wasm_bindgen]
+pub fn get_all_date_recurrences(rule_set_string: &str, limit: Option<u16>) -> Vec<JsValue> {
```

Comment:
Can we try to mimic the API for the [rrule.js library](https://github.com/jakubroztocil/rrule)? I think that would make it much simpler for users to swap out `rrule.js` for this wasm library.

Reply:
Sure I think it's a good idea